service/worker/deletenamespace/deleteexecutions/activities.go

```diff
@@ -49,6 +52,12 @@ type (
 		logger            log.Logger
 	}
 
+	LocalActivities struct {
```

Comment:
I don't get why do you want to separate them?

Reply:
Local activities execute on the workflow worker / tq and so they need to be registered there. The new activity task queues are only used for remote activities. I separated them to prevent registering all activities on both so it is clearer where everything will be executed.